command/genesis/params.go

```diff
@@ -38,7 +41,7 @@ var (
 	errValidatorsNotSpecified         = errors.New("validator information not specified")
 	errValidatorsSpecifiedIncorrectly = errors.New("validator information specified through mutually exclusive flags")
 	errUnsupportedConsensus           = errors.New("specified consensusRaw not supported")
-	errMissingBootnode                = errors.New("at least 1 bootnode is required")
+	errMissingBootnode                = errors.New("at least 1 BootNode is required")
```

Comment:
nitpick: revert `bootNode` back to `bootnode`

Reply:
+1

Reply:
Fixed a91c866e0f6d742a0cc4af63f07196b4703ec96a